client/localization/elements/sc-suttaplex/pl.json

```diff
@@ -1,11 +1,11 @@
 {
   "pl": {
-    "more": "Więcej",
     "beginner": "Rekomendowane dla początkujących studentów",
     "intermediate": "Rekomendowane dla średniozaawansowanych studentów",
     "advanced": "Rekomendowane dla zaawansowanych studentów",
     "originalTitle": "Oryginalny tytuł",
     "suttaCentralID": "Identyfikator SuttaCentral",
+    "alternateText": "Numer alternatywny w wydaniu {book}:",
```

Comment:
How did you know that {book} comes after "wydaniu"? Are you secretly Polish? :D

Reply:
Google is all-knowing!